aiida/backends/djsite/db/migrations/__init__.py

```diff
@@ -690,116 +715,73 @@ def create_value(self, key, value, subspecifier_value=None,
 
         :return: always a list of class instances; it is the user
           responsibility to store such entries (typically with a Django
-          bulk_create() call).
-        """
+          bulk_create() call)."""
+
         cls = self._model_class
         import datetime
 
         from aiida.common import json
         from aiida.common.timezone import is_naive, make_aware, get_current_timezone
 
+        other_attribs = other_attribs if other_attribs is not None else {}
+
         if self._subspecifier_field_name is None:
             if subspecifier_value is not None:
-                raise ValueError('You cannot specify a subspecifier value for '
-                                 'class {} because it has no subspecifiers'
-                                 ''.format(cls.__name__))
+                raise ValueError(
+                    'You cannot specify a subspecifier value for '
+                    'class {} because it has no subspecifiers'
+                    ''.format(cls.__name__)
+                )
             new_entry = cls(key=key, **other_attribs)
         else:
             if subspecifier_value is None:
-                raise ValueError('You also have to specify a subspecifier value '
-                                 'for class {} (the {})'.format(cls.__name__,
-                                                                self._subspecifier_field_name))
+                raise ValueError(
+                    'You also have to specify a subspecifier value '
+                    'for class {} (the {})'.format(cls.__name__, self._subspecifier_field_name)
+                )
             further_params = other_attribs.copy()
-            further_params.update({self._subspecifier_field_name:
-                                       subspecifier_value})
+            further_params.update({self._subspecifier_field_name: subspecifier_value})
             new_entry = cls(key=key, **further_params)
 
         list_to_return = [new_entry]
 
-        if value is None:
-            new_entry.datatype = 'none'
-            new_entry.bval = None
-            new_entry.tval = ''
-            new_entry.ival = None
-            new_entry.fval = None
-            new_entry.dval = None
-
-        elif isinstance(value, bool):
-            new_entry.datatype = 'bool'
-            new_entry.bval = value
-            new_entry.tval = ''
-            new_entry.ival = None
-            new_entry.fval = None
-            new_entry.dval = None
+        new_entry.datatype = 'none'
```

Comment:
this way, information is split over this function and `set_basic_data_attributes`.
I suggest you simply do all of this inside the function

Reply:
same as above.